scripts/drop-user-related-columns.sql

```diff
@@ -0,0 +1,8 @@
+DELETE FROM domains;
+DELETE FROM groups;
+DELETE FROM scaling_groups;
+DELETE FROM sgroups_for_domains;
+DELETE FROM users;
+DELETE FROM association_groups_users;
+DELETE FROM keypair_resource_policies;
+DELETE FROM keypairs;
```

Comment:
`scripts/drop-user-related-columns.sql` does not match with what it actually does.
If it's added for your local debugging, please remove it.